File: c#/1036.cs
using System;
using System.Globalization;

class URI {

    static void Main(string[] args) {

        string input_abcd = Console.ReadLine();

        string[] abc = input_abcd.Split(' ');

        double a = double.Parse(abc[0], CultureInfo.InvariantCulture);
        double b = double.Parse(abc[1], CultureInfo.InvariantCulture);
        double c = double.Parse(abc[2], CultureInfo.InvariantCulture);

        double delta = b * b - 4 * a * c;
                
        if ((a == 0) || (delta < 0)){
            Console.WriteLine("Impossivel calcular");
        } else {
            double x1 = (- b + Math.Sqrt(delta)) / (2 * a);
            double x2 = (- b - Math.Sqrt(delta)) / (2 * a);

            Console.WriteLine($"R1 = {x1.ToString("F5", CultureInfo.InvariantCulture)}");
            Console.WriteLine($"R2 = {x2.ToString("F5", CultureInfo.InvariantCulture)}");
        }
    }    
}

File: c++/1036.cpp
#include <stdio.h>
#include <math.h>

int main() {

    double a, b, c, delta, x1, x2;
    char value[50];

    fgets(value, sizeof(value), stdin);

    sscanf(value, "%lf %lf %lf", &a, &b, &c);
            
    delta = b * b - 4 * a * c;
                
    if ((a == 0) || (delta < 0)){
        printf("Impossivel calcular\n");

    } else {
        x1 = (- b + sqrt(delta)) / (2 * a);
        x2 = (- b - sqrt(delta)) / (2 * a);

        printf("R1 = %.5lf\n", x1);
        printf("R2 = %.5lf\n", x2);
    }
 
    return 0;
}